src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/OffenderDetailsUtils.kt

```diff
@@ -101,7 +102,7 @@ fun CaseSummary.asOffenderDetailSummary() = OffenderDetailSummary(
     riskColour = null,
     disabilities = null,
     genderIdentity = this.profile?.genderIdentity,
-    selfDescribedGender = null,
+    selfDescribedGender = this.profile?.selfDescribedGender,
```

Comment:
is this change needed for our stories? if not, then i would recommend to leave it as it was before.

Reply:
I set it as this value is coming from the profile like genderIdentity